lib/blog.js

```diff
@@ -45,6 +47,24 @@ class BlogPost {
   }
 
   _getRawMarkdown () {
+    // This block looks not very good, but they give confidence,
```

Comment:
This file has changed significantly since I was actively involved in the project. What is the purpose of the cache? It seems like this class could be simplified by not using this custom cache, instead using memory as the cache and loading the blog posts at startup/runtime.

Reply:
I think this PR: #2160 can give some feedback around the changes to the blog page handling.